src/main/kotlin/net/atos/zac/signalering/SignaleringService.kt

```diff
@@ -49,17 +55,22 @@ import java.util.logging.Logger
 class SignaleringService @Inject constructor(
     private val entityManager: EntityManager,
     private val drcClientService: DrcClientService,
+    private val restInformatieobjectConverter: RestInformatieobjectConverter,
     private val eventingService: EventingService,
     private val flowableTaskService: FlowableTaskService,
     private val mailService: MailService,
     private val signaleringenMailHelper: SignaleringMailHelper,
     private val zrcClientService: ZrcClientService,
-    private val restZaakOverzichtConverter: RestZaakOverzichtConverter
+    private val restZaakOverzichtConverter: RestZaakOverzichtConverter,
+    private val loggedInUserInstance: Instance<LoggedInUser>
 ) {
     companion object {
         private val LOG = Logger.getLogger(SignaleringService::class.java.name)
     }
 
+    private fun Instance<LoggedInUser>.getSignaleringZoekParameters() =
```

Comment:
Wij introduce a function for something so simple?

Reply:
That was a change requested with the initial Kotlin conversion :) We can revert this back but as this fragment was repeating several time in the code base, I think now it makes more sense and is self-explanatory what/why it does